src/components/Dashboard/TodoList.vue

```diff
@@ -79,8 +76,8 @@
                   <span>PAYMENT INCOMPLETE</span>
                 </div>
 
-                <!-- draft alteration not in good standing -->
-                <div v-else-if="isStatusDraft(item) && isTypeAlteration(item) && !item.goodStanding"
+                <!-- draft alteration to a BEN not in good standing -->
+                <div v-else-if="isStatusDraft(item) && isTypeAlteration(item) && isAlteringToBen && !isGoodStanding"
```

Comment:
This section is actually specific to alter-to-BEN filings but was incorrectly displaying for BC/CCC/ULC alterations.